_resourceexample/brainly.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Brainly Style Guide
+link: https://styleguide.brainly.com
+image: brainly.png
+tags:
+- frontend
+- code
+- javascript
+- react
+- html
+- css
+- sass
```

Comment:
I went through other entries and copied all matching tags - let me know if I used too many.